back/app/routes.py

```diff
@@ -1,10 +1,48 @@
+
 from app import app,db 
 from flask import jsonify, request
 
+
 @app.route('/')
 def home():
     return "Hello, Flask!"
 
+
+@app.route('/create',methods = ['POST'])
+def create():
+        try:
+            data = request.get_json()
+            name = data["name"]
+            age  = data["age"]
+            level= data["level"]
+            msg  = data["msg"]
+            # # عشان اتأكد ان الداتا اللى واصلانى وصلانى صح 
+            # if name is None or name =="" or not isinstance(name, str):
+            #     return ValueError('Invalid "name" field')
+            # if age is None or not isinstance(age, data):
+            #     return ValueError('Invalid "age" field')
+            # if level is None or not level in ["easy","normal","hard"]:
+            #     return ValueError('Invalid "level" field')
+            # if msg is None or msg == "" or not isinstance(msg, str):
+            #     return ValueError('Invalid "msg" field')
+            # # if important is None or not important in ["true","false"]:
+            # #     return ValueError('Invalid "important" field')
+            # # if status is None or not status in ["open","close"]:
+            # #     return ValueError('Invalid "status" field')
+            result = db.data.insert_one(data)
```

Comment:
comment this line until the serializer is made. so the database is not effected badly with wrong ObjectId